packages/tests/server/websockets.test.ts

```
@@ -56,10 +63,9 @@ function factory(config?: {
       return `hello ${input ?? 'world'}`;
     }),
 
-    slow: t.procedure.mutation(async ({}) => {
+    mut: t.procedure.mutation(async ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)